src/NuGet.Indexing/NuGetQueryParser.cs

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NuGet.Indexing
+{
+    public class NuGetQueryParser
+    {
+        private readonly Dictionary<QueryField, string[]> _queryFieldNames = new Dictionary<QueryField, string[]>
+        {
+            {QueryField.Id,  new [] { "Id" }},
```

Comment:
no.. changed to lower case

---
In reply to: [64862493](https://github.com/NuGet/NuGet.Services.Metadata/pull/73#discussion_r64862493) [](ancestors = 64862493)